recipes/nvidia.rb

```diff
@@ -119,7 +119,11 @@
 log 'yum makecache cuda' do
   message 'yum makecache cuda'
   only_if { ::File.exist?(makecache_file) }
+end
```

Comment:
Shouldn't this log resource go away and be replaced by the notify_group resource below?

Reply:
I kept it because it was printing a message to the log. Do you want me to remove it?

Reply:
Yes please.